sqltocsv.go

```diff
@@ -32,21 +44,26 @@ func main() {
 		log.Fatal(err)
 	}
 
-	output, err := os.Create(cfg.outputFile)
-	if err != nil {
-		log.Fatal(err)
+	csvWriter := csv.NewWriter(os.Stdout)
+	// If I ever need to support more than tabs/commas, this needs improving
+	if cfg.delimeter != "comma" {
+		if comma, ok := delimeters[cfg.delimeter]; ok {
+			csvWriter.Comma = comma
+		} else {
+			log.Printf("Warning: No known delimeter for %s, defaulting to Comma", cfg.delimeter)
+		}
 	}
 
-	csvWriter := csv.NewWriter(output)
-	csvWriter.Comma = 0x0009
-	firstLine := true
+	converter := converters.GetConverter(cfg.dbAdapter)
+
+	count := 0
```

Comment:
I don't do that by design. I do not like that zero value nonsense for primitives.